app/assets/stylesheets/color_themes/_color_theme_override_origwhite.scss

```diff
@@ -1,5 +1,5 @@
 body {
-  #main_stream .stream_element {
```

Comment:
- Avoid using id selectors
- Selector `main_stream` should be written in lowercase with hyphens